scripts/change_url

```diff
@@ -7,9 +7,9 @@ ynh_systemd_action --service_name=$app --action="stop" --log_path="/var/log/$app
 
 ynh_change_url_nginx_config
 
-ynh_backup_if_checksum_is_different --file="$install_dir/coin/settings_local.py"
-ynh_render_template ../conf/local.py.j2 "$install_dir/coin/settings_local.py"
-ynh_store_file_checksum --file="$install_dir/coin/settings_local.py"
+prefix="${path#"/"}/"
+prefix=${prefix%"/"}
```

Comment:
Sounds like it could be simplified to

```suggestion
prefix="${path#"/"}"
```

but not sure, bash syntax is pretty obscure